cabinets/__init__.py

```diff
@@ -148,3 +148,17 @@ def delete(uri: Union[str, Path], **kwargs: Any):
     """
     cabinet_, path = from_uri(uri)
     return cabinet_.delete(path, **kwargs)
+
+
+def list(directory_uri: Union[str, Path], **kwargs: Any) -> List[str]:
```

Comment:
Feelings about defaulting to the current directory?

Reply:
This makes sense for the file system cabinet, but what about things like s3? You can't really list all buckets in the world...

Reply:
I guess this only makes sense for local file systems